src/main/java/hudson/plugins/sonar/action/SonarBuildBadgeActionFactory.java

```diff
@@ -41,6 +42,9 @@
    * The badge will have an URL if there aren't multiple URLs.
    */
   public Collection<? extends Action> createFor(Run run) {
+    if (run == null) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this condition so that it does not always evaluate to "false" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2583)